CVLByExample/TransientStorage/certora/specs/Mutexer.spec

```diff
@@ -20,6 +20,9 @@ hook ALL_TLOAD(uint loc) uint v {
     }
 }
 
+invariant lockStatusDontChange()
+    !contract_lock_status;
+
```

Comment:
given this invariant, isn't checkContractLock redundant?

Reply:
Yep removed